### DISCUSSION TITLE
레벨4 1차 주간회의

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#999] [General]
## 공통으로 할 일
- [ ] PR 정리
  - 일요일까지 진행중인 PR 마무리 할 수 있도록 안내
  - 남겨진 PR은 클로즈 후 필요한 경우 담당자가 다시 등록하도록 안내
- [ ] 이슈 정리
  - 자신이 작성했거나 어사인 되어있는 이슈를 확인한다
  - 완료되거나 필요없어진 경우 close
  - 유효한 이슈면 PROLOG 프로젝트에 등록
  - Status는 Backlog로 등록
  - 라벨 성격에 맞춰 등록
  - 마일스톤은 제거
- [ ] 안내 사항 공유
  - 프로젝트 칸반 보드 안내
    - 라벨, 마일스톤 등
  - 브랜치 전략 및 배포 전략 공유

**Comment by gracefulBrown** [comment]
## 사용성 개선 스쿼드
- [ ] 백로그 이슈를 포함하여 피쳐를 뽑아내기
- [ ] 피쳐 중 우선순위를 선정
- [ ] 1주일 단위로 나눠서 이슈 등록
- [ ] 우선순위 높은 순으로 `사용성 개선` 스쿼드 등록, 마일스톤 등록
- [ ] 담당자 할당 및 Due Date 등록

**Comment by pocojang** [comment]
## 기술 개선 스쿼드

- [ ] 문서화
  - [ ] 모니터링
  - [ ] 디자인 시스템
- [ ] 인수 인계 계획
- [ ] 기술 블로그
- [ ] 테스트 개선
